### PR TITLE
Add automatic unofficial schedule warning

### PR DESCRIPTION
**Pull request by powe97**

### CODE REVIEW

File: site/src/components/UnofficialScheduleWarning.vue [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        style="
+          text-align: center;
+          padding-top: 0.5rem;
+          margin-left: 1rem;
+          margin-right: 1rem;
+          padding-bottom: 1rem;
+          font-size: 1.7rem;
+          border-style: solid;
+          border-width: 1px;
+          color: #ff1111;
```
Comment (benjaminrsherman) [inline site/src/components/UnofficialScheduleWarning.vue:15]: Can this be set by the theme?

File: scrapers/sis_scraper/main.py [hunk 2]
```diff
@@ -90,7 +91,7 @@ async def get_section_information(section_url):
     return section_dict
 
 
-async def get_class_information(class_url):
+async def get_class_information(class_url, registration_dates):
```
Comment (benjaminrsherman) [inline scrapers/sis_scraper/main.py:94]: Why is this using an out-param instead of returning two values?

File: scrapers/sis_scraper/main.py [hunk 3]
```diff
@@ -211,14 +217,16 @@ async def get_classes_with_code(term, code):
         return await request.text()
 
 
-async def scrape_subject(term, name, code):
+async def scrape_subject(term, name, code, registration_dates):
```
Comment (benjaminrsherman) [inline scrapers/sis_scraper/main.py:220]: Why is this using an out-param instead of returning two values?

**Review by benjaminrsherman — CHANGES_REQUESTED** [review]

File: site/src/components/UnofficialScheduleWarning.vue (hunk 1 above)
Comment (powe97) [reply site/src/components/UnofficialScheduleWarning.vue]: Yes, see latest commit.

File: scrapers/sis_scraper/main.py (hunk 3 above)
Comment (powe97) [reply scrapers/sis_scraper/main.py]: Changed in latest commit to use multiple return values.

File: scrapers/sis_scraper/main.py (hunk 2 above)
Comment (powe97) [reply scrapers/sis_scraper/main.py]: Changed in latest commit to use multiple return values.

**Review by elihschiff — APPROVED** [review]

**Review by benjaminrsherman — APPROVED** [review]

**Pull request merged**